src/components/Main/Main.js

```diff
@@ -9,11 +9,12 @@ import Covid from "../Covid/Covid";
 import OrderList from "../Production/Production";
 import Calender from "../calender/Calender";
 import Footer from "../Footer/Footer"
-import RichestPerson from "../RichestPerson/RichestPerson"
-import Population from "../Population/Population"
-import WorldMap from "../Worldmap/WorldMap"
-import Converter from "../ExchangeRate/Converter"
-
+import {Donutchart} from "../Donut/Donutchart" ;
+import {Linechart} from "../Line/Linechart" ;
+import { Quotes } from "../Quote/Quotes";
```

Comment:
format imports

Reply:
Sir , this imports from external libraries